python/llm/src/bigdl/llm/transformers/__init__.py

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from .convert import ggml_convert_int4
+from .model import AutoModelForCausalLM, AutoModel
+from transformers import AutoTokenizer
```

Comment:
do we need this? we should not unconditionally depends `transformer` - import it only when needed